Trailheads/src/classes/AccountManager.cls

```diff
@@ -0,0 +1,12 @@
+@RestResource(urlMapping = '/Accounts/*/contacts')
+global with sharing class AccountManager {
+
+    @HttpGet
+    global static Account getAccount() {
+        RestRequest request = RestContext.request;
+        String AccountId = request.requestURI.substringBetween('Accounts/', '/contacts');
+        Account result = [SELECT Name, Id, (SELECT Id, Name FROM Contacts) FROM Account WHERE Id = :AccountId];
```

Comment:
pytanie za 100 punktów - co się stanie jak strzelę restem pod .../Acounts/dupa/contacts?

Reply:
I jeśli to jest z trailheadów - spoko, ale wymagam od teraz, żebyś się pozastanawiał nad kodem który kopiujesz. Celem trailheadów nie jest nauczyć Cię robić wszystko od razu dobrze, ale czasem są takie rzeczy jak tutaj, które aż się proszą o validację (na przykłąd). Zastanawiaj się zawsze czy i dlaczego to pierdolnie i jak temu zapobiec